starters/vue3-xstate-css/src/tests/GreetView.cy.ts

```diff
@@ -0,0 +1,36 @@
+import GreetView from '../views/GreetView.vue';
+
+describe('<GreetView />', () => {
+	it('renders', () => {
+		cy.mount(GreetView, { props: { query: 'from This Dot Labs!' } });
+	});
+
+	it('should display a message with my greeting when I pass in a greeting', () => {
+		cy.mount(GreetView, { props: { query: 'from Cypress!' } });
+		cy.get('[data-cy=message-result]').should(
+			'contain.text',
+			'Hello, from Cypress!'
+		);
+	});
+
+	it('should display a default message when no greeting is passed', () => {
+		cy.mount(GreetView, { props: { query: '' } });
```

Comment:
Managed to achieve this using Vue 3's provide and inject functionality. So we provide a value from the root of the app, but set a default empty string value when it's injected so if we didn't provide a value it will use an empty string instead.